google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -90,49 +92,53 @@ def sample_table_to_csv():
         return self.to_dataframe().to_csv(index=False)
 
 
-def _table_wrapper_from_documentai_table(
-    documentai_table: documentai.Document.Page.Table, text: str
-) -> Table:
-    r"""Returns a Table.
+@dataclasses.dataclass
+class Token:
+    """Represents a wrapped documentai.Document.Page.Token.
 
-    Args:
-        documentai_table (documentai.Document.Page.Table):
-            Required. A documentai.Document.Page.Table.
+    Attributes:
+        documentai_token (google.cloud.documentai.Document.Page.Token):
+            Required. The original google.cloud.documentai.Document.Page.Token object.
         text (str):
-            Required. UTF-8 encoded text in reading order
-            from the document.
-
-    Returns:
-        Table:
-            A Table.
-
+            Required. UTF-8 encoded text.
     """
 
-    header_rows = _table_rows_from_documentai_table_rows(
-        table_rows=list(documentai_table.header_rows), text=text
-    )
-    body_rows = _table_rows_from_documentai_table_rows(
-        table_rows=list(documentai_table.body_rows), text=text
-    )
-
-    return Table(
-        documentai_table=documentai_table, body_rows=body_rows, header_rows=header_rows
-    )
+    documentai_token: documentai.Document.Page.Token
+    text: str
 
 
 @dataclasses.dataclass
-class Block:
-    """Represents a wrapped documentai.Document.Page.Block.
+class Line:
+    """Represents a wrapped documentai.Document.Page.Line.
 
     Attributes:
-        documentai_block (google.cloud.documentai.Document.Page.Block):
-            Required. The original google.cloud.documentai.Document.Page.Block object.
+        documentai_line (google.cloud.documentai.Document.Page.Line):
+            Required. The original google.cloud.documentai.Document.Page.Line object.
         text (str):
             Required. UTF-8 encoded text.
     """
 
-    documentai_block: documentai.Document.Page.Block
+    documentai_line: documentai.Document.Page.Line
     text: str
+    tokens: List[Token]
```

Comment:
yea users should never wrap their own `Line,Entity,Paragraph,etc` they should only use the Document.from_ to wrap a whole document not specific parts of a document.  My thought process if they are lazy calculated we'll need to pass in the whole page to be able to access things. I think for the initial release this might be okay and if there are performance issues we can go and refactor to do lazy loading.

Reply:
Sounds good on not worrying about lazy calculation until later.

On the other hand, let's evaluate whether the whole page should be passed to every element as a private attribute.  Only a reference of the page will be passed so there will be no copies.  As is right now, every time an element with layout needs to look up some text, the caller of the method has to pass in a page, since only the page has the text (is this true?).

There seems no harm to just include the page of an element (Block, Line, etc) as a private attribute of the element, and it would simplify the code (we will no longer need to pass in `text` in the methods).

This is assuming that every element belongs to only one page, though.  Do we know this is true?

Reply:
So it every element that is loaded is associated to a specific page so as long as we pass in the page i think we will be fine. I'll try out passing the page as reference.